tests/integration/suite/conftest.py

```diff
@@ -375,6 +375,26 @@ def clean():
     return _cleanup
 
 
+@pytest.fixture
+def remove_k8s_resource(admin_mc, request):
```

Comment:
The naming and comment of this should be updated to reflect that this only works with CRDs, not normal k8s objects.